### DISCUSSION TITLE
Rename master branch to main.

### DISCUSSION THREAD

**Discussion by ioquatix** [rack/rack#1798] [General]
I don't have a personal stake in this, but as a matter of consistency I've been renaming all my projects to use `main` rather than `master`.

Can we adopt this as a general policy for the rack organisation?

cc @tenderlove @jeremyevans

**Comment by jeremyevans** [comment]
I don't think the benefits are greater than the costs, but I don't object.

**Comment by ioquatix** [comment]
Okay, it is done.

**Comment by tenderlove** [comment]
I'm happy about this, but in the future please ping me first?  This discussion has only been up for less than 24hrs.

**Reply by ioquatix** [reply]
Sorry. Are you happy we do this for other projects under the rack org?

**Reply by matthewd** [reply]
@ioquatix  I think most of the other repositories are functionally maintained by other people? I too would prefer the consistency, but probably worth looping them in first.

**Reply by ioquatix** [reply]
That makes total sense.

**Reply by tenderlove** [reply]
Agree with @matthewd, I also like the consistency, but definitely loop them in.

**Comment by ioquatix** [comment]
Any objections to moving https://github.com/rack/rack-contrib `master` -> `main`?

**Reply by ioquatix** [reply]
Okay, done.

**Comment by ioquatix** [comment]
Any objections to moving https://github.com/rack/rack-test `master` -> `main`?

**Reply by ioquatix** [reply]
Okay, done.

**Comment by ioquatix** [comment]
I've opened a discussion on `rack-attack` which has active maintainers: https://github.com/rack/rack-attack/discussions/571

**Reply by ioquatix** [reply]
Okay, done.
